src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -82,7 +83,7 @@
         </service>
 
         <service id="resolve_controller_name_subscriber" class="Symfony\Bundle\FrameworkBundle\EventListener\ResolveControllerNameSubscriber">
```

Comment:
this service would need to be deprecated  as well

Reply:
There is definitely no way to deprecate such service

Reply:
can't we use the same trick as above, with the deprecated alias?